scripts/functional.js

```diff
@@ -17,12 +17,12 @@ const sandbox = require("@adobe/reactor-sandbox");
 const argv = require("minimist")(process.argv.slice(2));
 const chalk = require("chalk");
 
-const { watch } = argv;
+const { watch, testName: testNameFilter, src } = argv;
```

Comment:
I haven't tried fixture.only and test.only, this will test one test and/or one file at a time via the command line.  I'll add a comment.  I prefer to use the command line because then I don't forget and leave the only in the commit.

Reply:
I'm totally fine keeping this but I think we just need to/should add https://www.npmjs.com/package/eslint-plugin-no-only-tests to our lint rules. I'll create a ticket.

Reply:
I like that lint rule. Is there one for fdescribe, and ftest?
The --src option also gives us the ability to execute only a directory which is not possible with only.  (i.e. execute all the consent tests)